packages/vite-plugin-cloudflare/src/websockets.ts

```diff
@@ -1,4 +1,4 @@
-import ws from "ws";
+import * as ws from "ws";
```

Comment:
I think importing directly is clearer as we're only using one export.

```suggestion
import { WebSocketServer } from "ws";
```